torch/CMakeLists.txt

```diff
@@ -288,47 +188,60 @@ target_compile_options(torch
   -Wno-missing-braces # This warning is buggy
   -Wno-unknown-pragmas)
 
-if ($ENV{WERROR})
+if ($ENV{TORCH_WERROR})
```

Comment:
If this env var got renamed, doesn't the CI env var have to track the change? (Why rename it at all?)

Reply:
I did change it in CI around usages of cpp_build (explanation is by my comment above)

Reply:
This should be undone, see comment above.